core/test_network/Cargo.toml

```diff
@@ -20,11 +20,13 @@ iroha_actor = { version = "=2.0.0-pre-rc.5", path = "../../actor" }
 iroha = { path = "../../cli", features = ["test-network"] }
 
 eyre = "0.6.5"
+futures = { version = "0.3.17", default-features = false, features = ["std", "async-await"] }
```

Comment:
I'm assuming you sorted this alphabetically. It would be nice if we made it standard.